### PR TITLE
Milestone 4 - Desktop Version

### PR DESCRIPTION
**Pull request by chelmerrox**
I need a review for this project. I'm using the fifth template from Figma as my design guide.
I have also completed it & its requirements to:

- create the desktop version of the web page
- use a Formspree service for the form's action attribute
- have pseudoclasses for the form's submit button
- stick to the styling of the template I chose
- use flex & grid for positioning elements

Thanks in advance for your review, comments & help!

### CODE REVIEW

File: styles/my-portfolio.css [hunk 1]
```diff
@@ -595,4 +625,868 @@ body {
...
+  .image-11-desktop,
+  .image-8-desktop {
+    grid-row-start: 1;
+    grid-row-end: 2;
+  }
+
+  .image-4-desktop {
+    grid-column-start: 1;
+    grid-column-end: 2;
+    background-image: url('../images/image4.jpg');
```
Comment (FrederickMih) [inline styles/my-portfolio.css:964]: - [ ] Please let's adjust these images to match the original design. Screenshots below for visual perspective

Your Implementation | Figma Design
:---:|:---:
 |

File: my-portfolio.html [hunk 2]
```diff
@@ -374,8 +708,102 @@ <h2 class="playfair-display-font font-color get-started-title">Get started</h2>
...
+            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <g clip-path="url(#clip0_67_1253)">
+                <path d="M24 4.5585C23.1075 4.95 22.1565 5.2095 21.165 5.3355C22.185 4.7265 22.9635 3.7695 23.3295 2.616C22.3785 3.183 21.3285 3.5835 20.2095 3.807C19.3065 2.8455 18.0195 2.25 16.6155 2.25C13.8915 2.25 11.6985 4.461 11.6985 7.1715C11.6985 7.5615 11.7315 7.9365 11.8125 8.2935C7.722 8.094 4.1025 6.1335 1.671 3.147C1.2465 3.8835 0.9975 4.7265 0.9975 5.634C0.9975 7.338 1.875 8.8485 3.183 9.723C2.3925 9.708 1.617 9.4785 0.96 9.117C0.96 9.132 0.96 9.1515 0.96 9.171C0.96 11.562 2.6655 13.548 4.902 14.0055C4.5015 14.115 4.065 14.1675 3.612 14.1675C3.297 14.1675 2.979 14.1495 2.6805 14.0835C3.318 16.032 5.127 17.4645 7.278 17.511C5.604 18.8205 3.4785 19.6095 1.1775 19.6095C0.774 19.6095 0.387 19.5915 0 19.542C2.1795 20.9475 4.7625 21.75 7.548 21.75C16.602 21.75 21.552 14.25 21.552 7.749C21.552 7.5315 21.5445 7.3215 21.534 7.113C22.5105 6.42 23.331 5.5545 24 4.5585Z" fill="white"/>
+              </g>
+              <defs>
+                <clipPath>
+                  <rect width="24" height="24" fill="white"/>
```
Comment (FrederickMih) [inline my-portfolio.html:806]: - [ ] These icons are not showing up in the footer section as seen in the Figma design. Please let's fix this.
- [ ] Also make sure that `Get my CV` is aligned as in Figma design.
- [ ] **Also make sure that you get rid of the horizontal scrolling**. It's not good for the user experience. People expect to scroll vertically for additional content, but they don't expect to scroll sideways.

Your Implementation | Figma Design
:---:|:---:
 |

File: my-portfolio.html [hunk 3]
```diff
@@ -322,9 +564,101 @@ <h2 class="playfair-display-font font-color get-started-title">Get started</h2>
...
+            </pattern>
+            <pattern patternContentUnits="objectBoundingBox" width="1.47113" height="1.47077">
+              <use xlink:href="#image0_67_825" transform="scale(0.00570207)"/>
+            </pattern>
+          </defs>
+        </svg>
+
+        <!--Get Started text & Contact Form-->
+        <section class="get-started-form">
+          <h2 class="playfair-display-font get-started-title">Get started</h2>
```
Comment (FrederickMih) [inline my-portfolio.html:601]: - [ ] Please make sure that the font, width, height, margins of this header is the same as that in the original design

File: my-portfolio.html [hunk 4]
```diff
@@ -322,9 +564,101 @@ <h2 class="playfair-display-font font-color get-started-title">Get started</h2>
...
+          
+          <!--Get Started Paragraph-->
+          <p class="poppins-font">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+          
+          <!--Contact Form-->
+          <form action="https://formspree.io/f/xzboarnl" method="POST" name="contact-form" class="contact-form">
+            <div class="name-email-section">
+              <!--Name-->
+              <div class="name-section">
+                <label for="name-desktop" class="poppins-font user-name-label">Your name</label>
```
Comment (FrederickMih) [inline my-portfolio.html:611]: - [ ] As seen in Figma design, the label **Your name** is outside the field. Please do the same for **Your email** and **Message**

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @chelmerrox  , 

You have done a fantastic job so far in this project. Keep up! :rocket: 
There are some issues that you still need to work on to go to the next project but you are almost there!



# To Highlights

-  Git/GitHub workflow has been followed :heavy_check_mark: 
-  Linters are correctly set up :heavy_check_mark: 
-  Pull Request has a title and summary :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles/my-portfolio.css [hunk 5]
```diff
@@ -595,4 +625,921 @@ body {
...
+
+  .get-started-desktop {
+    display: grid;
+    grid-template-columns: 1fr 3fr 1fr;
+    grid-template-rows: 1fr;
+    background-color: #e5e5e5;
+    width: 1400px;
+    margin-top: 100px;
+    padding-bottom: 45px;
+  }
```
Comment (keroloussamy) [inline styles/my-portfolio.css:1351]: - [ ] Kindly remove the `width: 1400px;`, to make your design perfect.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @chelmerrox, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

 ## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Omar-Muhamad — APPROVED** [review]
# Approved ✔
### Hi @chelmerrox,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**